server/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/Db2SqlLexer.g4

```diff
@@ -544,91 +544,16 @@ SPECIAL : S P E C I A L;
 SPECIFIC : S P E C I F I C;
 SQLADM : S Q L A D M;
 SQLCA : S Q L C A;
```

Comment:
dbs_include with member can be in different regions. So, I can create new  dbs_include_sqlca_da which can replace both dbs_include_sqlca and dbs_include_sqlda, and I can remove those SQLCA/DA in lexer.

Reply:
> Can we use `dbs_include` instead of `dbs_include_sqlca` and `dbs_include_sqlda`? In that case we can also delete `SQLCA` and `SQLDA` lexers.

I tried it but usages are different in sections, so we have to keep it as is.